internal/pkg/admission/policy-server-deployment_test.go

```diff
@@ -125,3 +135,67 @@ func createDeployment(replicasInt int, serviceAccount, imagePullSecret, image st
 
 	return policyServerDeployment
 }
+
+func insertContainer(deployment *appsv1.Deployment) {
+	container := corev1.Container{
+		Name:  "container0",
+		Image: "container0image:latest",
+	}
+	containers := []corev1.Container{container}
+	for index := range deployment.Spec.Template.Spec.Containers {
+		containers = append(containers, deployment.Spec.Template.Spec.Containers[index])
+	}
```

Comment:
```suggestion
		containers = append(containers, deployment.Spec.Template.Spec.Containers...)
```
nit: we don't need the fork to add the containers to the slice